relay-general/src/store/normalize.rs

```diff
@@ -555,6 +584,7 @@ pub fn light_normalize_event(
         )?; // Timestamps are core in the metrics extraction
         normalize_event_tags(event)?; // Tags are added to every metric
         normalize_exceptions(event)?; // Browser extension filters look at the stacktrace
+        light_normalize_stacktraces(event)?;
```

Comment:
`light_normalize_stacktraces` doesn't replace `normalize_exceptions`, as it works with stacktraces. The operation is simple and it's possible to refactor the method and keep only the stuff we need in light normalization, but I don't want to refactor that until it becomes an issue or we need some performance improvement.